docs/apis-tools/frontend-development/introduction-to-task-applications.md

```diff
@@ -0,0 +1,141 @@
+import TaskAssignmentImg from './img/task-assignment.png';
+import TaskLifecycleImg from './img/task-lifecycle.png';
+import styles from "./styles.module.css";
+
+Task applications are the interface between humans and Camunda processes to orchestrate human work. Learn key concepts of the architecture of task applications before you build your own.
+
+:::note
+If you are not familiar with the idea of human task management itself, read the [introduction to human task management](/components/best-practices/architecture/understanding-human-tasks-management.md) first.
+:::
+
+## What are task applications?
+
+Task applications are end-user applications that allow humans to perform work orchestrated with a process. A [user task](/components/modeler/bpmn/user-tasks/user-tasks.md/#user-task-forms) (also referred to as a **human task**) represents a single **work item** to be performed by an individual or a group. The jobs of a task application include:
+
+- Listing available tasks and allowing users to select a task to work on.
+- Providing filter and search options for users so they can more easily find the right next task to work on.
+- Presenting the selected task and an interface for completing the task, usually via a form.
+- Providing an interface to create new tasks, e.g. by starting a new process.
+- Provide insight into the progress of work tasks, including processes and cases.
+- Aggregate information so users and their managers can assess the impact on process goals, such as KPIs and SLAs.
+- Ensure tasks are visible only to authorized users.
+
+Task applications play a key role in the orchestration of business processes. They enable the orchestration of processes that still contain manual work instead of automating each process step in advance. This unlocks the potential for continuous improvement and for identifying opportunities for process optimization and automation.
+
+:::tip
+Not sure if you should use Camunda Tasklist, build your custom task application, or use a third-party application? Read the [guide to understand human task management](/components/best-practices/architecture/understanding-human-tasks-management.md#deciding-about-your-task-list-frontend) first.
+:::
+
+## Tasklist layout
+
+Camunda 8 comes with a ready-to-use Tasklist UI that implements all key concepts of a task application. The Tasklist UI is a generic task application; your custom task application should probably be tailored to your specific use case and also include external data sources and tools.
+
+The Tasklist UI is split into two main pages: the [tasks page](#task-page) and the [processes page](#processes-page).
+
+### Task page
+
+The task page lists all tasks pending for a user or user group, and allows users to pick and claim a task from that queue to work on. On the same page, the details of a selected task are displayed including the form that the user must submit in order to execute and complete the task.
```

Comment:
Should we add some sentence talking about the possibility to unassign/assign a task?

Reply:
I will do this in a follow-up PR